src/web/src/components/Posts/Post.tsx

```diff
@@ -286,6 +286,51 @@ const extractBlogClassName = (url: string) => {
 // a 'guid' from a YouTube video is usually written as 'yt:video:id'
 const extractVideoId = (post: Post): string => post.guid.split(':')[2];
 
+/**
+ * Destroys all the .zoomed-image-container elements in the DOM
+ * @return {number} Number of elements destroyed
+ */
+const zoomOutAllImages = () => {
+  const zoomedImgContainers = document.querySelectorAll<HTMLDivElement>('.zoomed-image-container');
```

Comment:
```suggestion
  const zoomedImgContainers = PostElement.querySelectorAll<HTMLDivElement>('.zoomed-image-container');
```
Don't select all images from the DOM but just the current post.

The Logo button itself is an image too so you'll get this when click on it
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/51073515/154164752-dce4b23a-9f8b-4ac6-befd-49b2ca9e7314.png">

Reply:
Or this when you click on @TueeNguyen' avatar :)
<img width="1119" alt="image" src="https://user-images.githubusercontent.com/51073515/154166565-26dad31f-1116-4335-929b-803edc2ea46f.png">

Reply:
Why me 😂